authui/src/authflowv2/zxcvbn.ts

```diff
@@ -0,0 +1,10 @@
+// This file lazily imports zxcvbn, so that it's not included in the main bundle
+
+import type zxcvbn from "zxcvbn";
```

Comment:
I wanted to use eslint to ban import from `zxcvbn`, but I found that we did not lint authui 🥲 
Maybe I create a new issue for adding eslint.